### PR TITLE
Adding "fight" plots with only optimizers with all settings

### PR DESCRIPTION
**Pull request by teytaud**
## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] The documentation is up-to-date with the changes I made.
- [x] I have read the [**CONTRIBUTING**](https://facebookresearch.github.io/nevergrad/contributing.html) document and completed the CLA (see [**CLA**](https://facebookresearch.github.io/nevergrad/contributing.html#contributor-license-agreement-cla)).
- [x] All tests passed, and additional code has been covered with new tests.

### CODE REVIEW

**Review by jrapin — COMMENTED** [review]
I dont't get this + there are a mixture of different stuff, can you explain?

**Comment by teytaud** [comment]
> I dont't get this + there are a mixture of different stuff, can you explain?

Yes there are two modifications in the same PR, sorry:
- one of the modifications, in addition of plotting our current plots, adds a "pure" plot: it contains only algorithms which have been fully run. This is now used in the dashboard.
- the other modification creates text files with rankings. Convenient for inclusion in the dashboard.

File: nevergrad/benchmark/plotting.py [hunk 1]
```diff
@@ -638,6 +667,16 @@ def winrates_from_selection(
         num_rows = min(num_rows, len(all_optimizers))
         # iterate on all sub cases
         victories, total = aggregate_winners(df, categories, all_optimizers)
+        if restricted_to_complete:
+            print([int(2 * victories.loc[n, n]) for n in all_optimizers])
+            max_num = max([int(2 * victories.loc[n, n]) for n in all_optimizers])
+            new_all_optimizers = [n for n in all_optimizers if int(2 * victories.loc[n, n]) == max_num]
+            print(new_all_optimizers)
+            print("vs")
+            print(all_optimizers)
```
Comment (jrapin) [inline nevergrad/benchmark/plotting.py:676]: please clean the prints and add a dosctring to explain what "restricted_to_complete" does. Maybe "only_completed" is simpler and clearer?

**Review by jrapin — COMMENTED** [review]
See comment
I'm just a bit afraid that we start generating too many files and it becomes a mess. Let's do it for now and clean later.

File: nevergrad/benchmark/plotting.py (hunk 1 above)
Comment (teytaud) [reply nevergrad/benchmark/plotting.py]: Done.

**Review by jrapin — APPROVED** [review]

**Pull request merged**